Sources/XcodeProj/Project/XCSharedData.swift

```diff
@@ -55,4 +55,11 @@ public final class XCSharedData: Equatable {
             lhs.breakpoints == rhs.breakpoints &&
             lhs.workspaceSettings == rhs.workspaceSettings
     }
+
+    // MARK: - Writable
+
+    public func write(path: Path, override: Bool) throws {
+        try XcodeProj.writeSchemes(schemes: schemes, path: path, override: override)
+        try XcodeProj.writeBreakPoints(breakpoints: breakpoints, path: path, override: override)
```

Comment:
These are static funcs on `XcodeProj` because they need to be used by both `XCSharedData` and `XCUserData`. We could also create an extension on Array for the schemes

```
extension Array: Writable where Element == XCScheme {
    public func write(path: Path, override: Bool) throws {
        if override, path.exists {
            try path.delete()
        }

        try path.mkpath()
        for scheme in self {
            try scheme.write(path: path + "\(scheme.name).xcscheme",
                             override: override)
        }
    }
}
```

And do `schemes.write(path: path, override: override)` here.. but the breakpoints is more complicated. 

I would like to hear your thoughts on this.